transport/http_util.go

```diff
@@ -228,7 +228,7 @@ func div(d, r time.Duration) int64 {
 
 // TODO(zhaoq): It is the simplistic and not bandwidth efficient. Improve it.
 func timeoutEncode(t time.Duration) string {
-	if d := div(t, time.Nanosecond); d <= maxTimeoutValue {
+    if d := int64(t); d <= maxTimeoutValue {
```

Comment:
The original function call hides the implementation detail about what time.Nanosecond actually is.
It improves the readability and is consistent with the other part of this function.